docs/graphql/manual/queries/query-filters.rst

```diff
@@ -921,6 +943,45 @@ Fetch a list of articles rated more than 4 or published after "01/01/2018":
       }
     }
 
+
+.. note::
+
+  The ``_or`` operator expects an array of expressions as input. Passing an object to it will result in the
```

Comment:
```suggestion
  The ``_or`` operator expects an array of expressions as input. If you pass a single object with multiple keys, then the expression resolves to the ``AND`` of the individual key expressions.
```

Reply:
The example should make this obvious, spelling out the behaviour verbally has no real value as nobody is going to use this behaviour anyway. Just being aware that this will not work as expected with an understanding why it is so
 is good enough